src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -140,9 +140,9 @@ public function getDigits(string $key, string $default = '')
      *
      * @return int The filtered value
      */
-    public function getInt(string $key, int $default = 0)
+    public function getInt(string $key, int $default = 0): int
```

Comment:
this should be reverted, that's a BC breaking change

Reply:
thanks - surely I can remove that but is it a BC breaking even though composer.json has `"php": "^7.2.9"` as requirement?

Reply:
Yes it is: it breaks all child classes that would override this method, even on >= 7.2

Reply:
ah - right. Let me change it then

Reply:
done